src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/LearnerMatchService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Commands.Persistence;
+using SFA.DAS.EmployerIncentives.Commands.Services.LearnerMatchApi;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives;
+using SFA.DAS.EmployerIncentives.Domain.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.RefreshLearner
+{
+    internal class LearnerMatchService
+    {
+        private readonly ILearnerService _learnerService;
+        private readonly ILearnerDomainRepository _learnerDomainRepository;
+        private readonly ICollectionCalendarService _collectionCalendarService;
+
+        internal LearnerMatchService(ILearnerService learnerService, ILearnerDomainRepository learnerDomainRepository, ICollectionCalendarService collectionCalendarService)
+        {
+            _learnerService = learnerService;
+            _learnerDomainRepository = learnerDomainRepository;
+            _collectionCalendarService = collectionCalendarService;
+        }
+
+        internal async Task<Learner> RefreshLearner(Domain.ApprenticeshipIncentives.ApprenticeshipIncentive incentive)
```

Comment:
Can this method be called just Refresh or Get?  It's the LearnerService and so are dealing with Learner data.